jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/Content.java

```diff
@@ -457,6 +463,11 @@ static Chunk from(ByteBuffer byteBuffer, boolean last)
          */
         static Chunk from(ByteBuffer byteBuffer, boolean last, Runnable releaser)
         {
+            if (!byteBuffer.hasRemaining())
```

Comment:
I think it's more readable if you invert the `if`.

Reply:
Ok.